codefresh/resource_pipeline_test.go

```diff
@@ -57,6 +57,7 @@ func TestAccCodefreshPipeline_Concurrency(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "spec.0.concurrency", "1"),
 					resource.TestCheckResourceAttr(resourceName, "spec.0.branch_concurrency", "2"),
 					resource.TestCheckResourceAttr(resourceName, "spec.0.trigger_concurrency", "3"),
+					resource.TestCheckResourceAttr(resourceName, "spec.0.pack_id", "5cd1746617313f468d669045"),
```

Comment:
where are you getting this value from? 
the tests passed for you locally?

you should create a new configuration test that will take this packId value as an argument and apply it.

Reply:
@mattmelgard Wdyt?

Reply:
@todaywasawesome / @ziv-codefresh I updated the tests to what I think they should look like but I'm not entirely certain on how to run the tests. Running `make test` seems to pass, but a good portion of them are skipped from what I can tell and I don't really see any repo-level documentation regarding how to run them either. Let me know and thanks.